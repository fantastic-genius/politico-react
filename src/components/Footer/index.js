import React from 'react';

const Footer = () => (
  <footer>
    <span>Copyright@2019  @fantastic-genius (Andela Bootcamp Cycle41)</span>
  </footer>
);

export default Footer;
