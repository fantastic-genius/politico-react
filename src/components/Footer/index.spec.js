import React from 'react';
import { shallow } from 'enzyme';
import Footer from './index';

describe('<Footer />', () => {
  describe('Rendering', () => {
    it('should render Footer correctly in debug mode', () => {
        const component = shallow(<Footer debug/>);
        expect(component).toMatchSnapshot();
    });
  })

})

