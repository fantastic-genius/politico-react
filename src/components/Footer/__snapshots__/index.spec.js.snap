// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Rendering should render Footer correctly in debug mode 1`] = `
<footer>
  <span>
    Copyright@2019  @fantastic-genius (Andela Bootcamp Cycle41)
  </span>
</footer>
`;
